tests/legacy/system/test_edb_modeler.py

```diff
@@ -390,3 +390,24 @@ def test_duplicate(self):
         assert edbapp.modeler.primitives_by_layer["mid_gnd"]
         assert edbapp.modeler.primitives_by_layer["bot_gnd"]
         edbapp.close()
+
+    def test_layer_name(self):
+        edbapp = Edb()
```

Comment:
Please use below edb for testing. Avoid create a layout for every test.
edbapp = Edb(target_path, desktop_version)

Reply:
Okay i corrected it please check